tests/providers/amazon/aws/executors/ecs/test_ecs_executor.py

```diff
@@ -103,16 +105,16 @@ def mock_config():
 
 @pytest.fixture
 def set_env_vars():
```

Comment:
Here and many of the changes below:  I just moved the repeatedly reused prefix into a constant.  Lots of the uses were getting pushed to multiple lines because the prefix was so long and it also makes sense to make that DRY.